src/containers/Auth/ConfirmEmailPage.js

```diff
@@ -29,7 +29,7 @@ function ConfirmEmailPage(props) {
     confirmEmail(query).catch(() => {
       history.push('/login');
     });
-  }, [confirmEmail, history]);
+  }, [confirmEmail, history, query]);
```

Comment:
Did you add it again?
I think I mentioned it before
As I remember, when we add this, it sends confirmation request multiple times

Reply:
I had removed that.
but i think eslint run command add it back.
can u verify again?